recipe/windows_cmake_install_bindir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index 3fd9d70..9ce3f67 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -398,7 +398,7 @@ endif ()
+ # installation directories
+ if (OS_WINDOWS)
+   set (RUNTIME_INSTALL_DIR Bin)
+-  set (LIBRARY_INSTALL_DIR Lib)
++  set (LIBRARY_INSTALL_DIR Bin)
```

Comment:
Not totally sure that we need this patch as noted in this [comment]( https://github.com/conda-forge/gflags-feedstock/pull/4#issuecomment-292349781 ).